src/comm.cc

```diff
@@ -780,36 +781,41 @@ commCallCloseHandlers(int fd)
     }
 }
 
+/// sets SO_LINGER socket(7) option
+/// \param enabled -- whether linger will be active (sets linger::l_onoff)
+/// \param timeoutInSeconds -- how long to linger for (sets linger::l_linger)
+static void
+commConfigureLinger(const int fd, const OnOff enabled, const int timeoutInSeconds = 0)
+{
+    struct linger l;
```

Comment:
They probably do not exist, but let's initialize all fields, including the ones we might not know about:

```suggestion
    struct linger l = {};
```

I bet an optimizing compiler removes this initialization when `linger` has only two fields that we explicitly initialize below.

Reply:
Ok.